ckanext/odata/templates/ajax_snippets/api_info.html

```diff
@@ -138,10 +138,10 @@ <h3>
       <div id="collapse-python" class="accordion-body collapse">
         <div class="accordion-inner">
           <pre>
-import urllib
+from urllib import request
```

Comment:
I recommend changing this to `import urllib.request`. Users may be confuse this with the different `requests` module